types/passport-instagram/tslint.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please disable these rules in the source instead of globally, and even there only if necessary.

Reply:
@uniqueiniquity thanks for pointing out the issue. :-) I have removed all the unnecessary rules - just retained 2 rules that I need here. Do let me know if that's ok